integration-tests/src/tests/features/chunk_nodes_cache.rs

```diff
@@ -17,6 +14,10 @@ use near_primitives::version::{ProtocolFeature, ProtocolVersion};
 use near_primitives::views::FinalExecutionStatus;
 use near_store::trie::TrieNodesCount;
 
```

Comment:
Undo these changes, the prevailing code style is to not separate imports into groups.